src/WebJobs.Script.WebHost/Metrics/FlexConsumptionMetricsPublisher.cs

```diff
@@ -66,13 +68,13 @@ public FlexConsumptionMetricsPublisher(IEnvironment environment, IOptionsMonitor
 
         internal bool IsAlwaysReady { get; set; }
 
-        internal string MetricsFilePath { get; set; }
+        internal LegionMetricsFileManager MetricsFileManager => _metricsFileManager;
 
         public void Start()
         {
             Initialize();
 
-            _logger.LogInformation($"Starting metrics publisher (AlwaysReady={IsAlwaysReady}, MetricsPath='{MetricsFilePath}').");
+            _logger.LogInformation($"Starting metrics publisher (AlwaysReady={IsAlwaysReady}, MetricsPath='{_metricsFileManager.MetricsFilePath}').");
```

Comment:
Any reason we aren't using structured logging?

Reply:
No reason, other than I usually don't I guess. Gonna leave these as is for now.